[1.12.2-2.5a] Crash when Fission Reactor explodes

Whenever a Fission Reactor explodes, the game crashes with the following report :

```
[Server thread/ERROR]: Encountered an unexpected exception
net.minecraft.util.ReportedException: Ticking block entity
	at net.minecraft.server.MinecraftServer.func_71190_q(MinecraftServer.java:771) ~[MinecraftServer.class:?]
	at net.minecraft.server.MinecraftServer.func_71217_p(MinecraftServer.java:666) ~[MinecraftServer.class:?]
	at net.minecraft.server.integrated.IntegratedServer.func_71217_p(IntegratedServer.java:185) ~[chd.class:?]
	at net.minecraft.server.MinecraftServer.run(MinecraftServer.java:524) [MinecraftServer.class:?]
	at java.lang.Thread.run(Thread.java:745) [?:1.8.0_25]
Caused by: java.lang.IllegalArgumentException: Cannot get property PropertyDirection{name=facing, clazz=class net.minecraft.util.EnumFacing, values=[north, south, west, east]} as it does not exist in BlockStateContainer{block=minecraft:lava, properties=[level]}
	at net.minecraft.block.state.BlockStateContainer$StateImplementation.func_177229_b(BlockStateContainer.java:201) ~[awu$a.class:?]
	at nc.block.tile.generator.BlockFissionController.setState(BlockFissionController.java:54) ~[BlockFissionController.class:?]
	at nc.tile.generator.TileFissionController.setBlockState(TileFissionController.java:116) ~[TileFissionController.class:?]
	at nc.tile.generator.TileFissionController.updateGenerator(TileFissionController.java:101) ~[TileFissionController.class:?]
	at nc.tile.generator.TileItemGenerator.func_73660_a(TileItemGenerator.java:64) ~[TileItemGenerator.class:?]
	at net.minecraft.world.World.func_72939_s(World.java:1829) ~[amu.class:?]
	at net.minecraft.world.WorldServer.func_72939_s(WorldServer.java:612) ~[oo.class:?]
	at net.minecraft.server.MinecraftServer.func_71190_q(MinecraftServer.java:765) ~[MinecraftServer.class:?]
	... 4 more
```

Upon reloading the world Lava is correctly placed and the Reactor is properly destroyed.

--- a/src/main/java/nc/tile/generator/TileFissionController.java
+++ b/src/main/java/nc/tile/generator/TileFissionController.java
@@ -75,12 +75,15 @@ public class TileFissionController extends TileItemGenerator {
 				consume();
 			}
 		}
-			tickStructureCheck();
-			checkStructure();
+		tickStructureCheck();
+		checkStructure();
+
 		if(!world.isRemote) {
 			tickRunCheck();
 			run();
-			overheat();
+			if (overheat() == true) {
+				return;
+			}
 			if (canProcess() && isPowered()) {
 				isGenerating = true;
 				time += getRateMultiplier();
@@ -155,10 +158,12 @@ public class TileFissionController extends TileItemGenerator {
 		return false;
 	}
 	
-	public void overheat() {
+	public boolean overheat() {
 		if (heat >= getMaxHeat() && NCConfig.fission_overheat) {
 			meltdown();
+			return true;
 		}
+		return false;
 	}
 	
 	public void meltdown() {

